### ISSUE TITLE
Fireworks example exits when minimizing on Windows

### ISSUE THREAD

**Issue by ZargorNET** [vlang/v#9485]
Specs:
```
OS: windows, Microsoft Windows 10 Home v19042 64-bit
Processor: 12 cpus, 64bit, little endian, AMD Ryzen 5 5600X 6-Core Processor
CC version: Error: exec failed (CreateProcess) with code 2: The system cannot find the file specified.
 cmd: cc --version

getwd: C:\Users\Zargor\Desktop\Custom Shell Scripts\v\examples
vmodules: C:\Users\Zargor\.vmodules
vroot: C:\Users\Zargor\Desktop\Custom Shell Scripts\v
vexe: C:\Users\Zargor\Desktop\Custom Shell Scripts\v\v.exe
vexe mtime: 2021-03-27 14:30:21
is vroot writable: true
is vmodules writable: true
V full version: V 0.2.2 7664964

Git version: git version 2.25.1
Git vroot status: Error: fatal: not a git repository (or any parent up to mount point /mnt)
Stopping at filesystem boundary (GIT_DISCOVERY_ACROSS_FILESYSTEM not set).
.git/config present: false
thirdparty/tcc status: thirdparty-windows-amd64 95766bbe
```

**What did you do?**
Opened the fireworks example and minimized it. The application exits.

**What did you expect to see?**
Minimizing, but still running
 
**What did you see instead?**
Exiting

**Comment by JalonSolov** [comment]
@shadowninja55 ^^

**Comment by spytheman** [comment]
Should be fixed in f0e36ca.
